src/tokens/backgroundColor.js

```diff
@@ -39,7 +39,14 @@ const light = core => ({
   RadioDot: core.Color.Neutral100,
   RadioDotChecked: core.Color.Primary300,
   RadioDotDisabled: core.Color.Neutral200,
-  RadioDotDisabledChecked: core.Color.Neutral500
+  RadioDotDisabledChecked: core.Color.Neutral500,
+  DropdownLabelDefault: core.Color.NeutralWhite,
+  DropdownLabelSecondary: core.Color.Neutral100,
+  DropdownMenu: core.Color.NeutralWhite,
+  DropdownMenuItemHover: core.Color.Neutral100,
+  DropdownMenuItemActive: core.Color.Primary100,
+  DropdownChip: core.Color.Primary100,
```

Comment:
What do you think about naming this InputChip so we can have a clear distinction about the input and the dropdown portion.

Reply:
👍

Reply:
We need a backgroundColor for our dropdown menus like the actions menu and the user menu. Are they the same color as this dropdown? I assume this one is for our select dropdown.

Reply:
They should be the same color @jgerlach

Reply:
awesome. Thanks! We also need to add a backgroundColor for the close button used in the Modal and Drawer components